app/src/main/res/layout-sw360dp/list_item_stream.xml

```diff
@@ -75,6 +75,13 @@
                 android:layout_width="match_parent"
                 android:layout_height="230dp"
                 android:scaleType="centerCrop" />
+            <View
```

Comment:
This view has to be added on the layout/ folder file, also so it doesn’t crash on a NullPointerException